algobot/__main__.py

```diff
@@ -407,18 +410,19 @@ def end_backtest(self):
         """
         Ends backtest and prompts user if they want to see the results.
         """
-        backtestFolderPath = create_folder('Backtest Results')
-        innerPath = os.path.join(backtestFolderPath, self.backtester.symbol)
-        create_folder_if_needed(innerPath, backtestFolderPath)
-        defaultFile = os.path.join(innerPath, self.backtester.get_default_result_file_name())
-        fileName, _ = QFileDialog.getSaveFileName(self, 'Save Result', defaultFile, 'TXT (*.txt)')
-        fileName = fileName.strip()
-        fileName = fileName if fileName != '' else None
+        backtest_folder_path = helpers.PATHS.get_backtest_results_dir()
+        create_folder_if_needed(backtest_folder_path)
+        inner_path = os.path.join(backtest_folder_path, self.backtester.symbol)
```

Comment:
We should probably refactor this code to use makedirs()

Reply:
`create_folder_if_needed` calls that under the hood.